aptos-move/framework/aptos-framework/sources/PropertyList.move

```diff
@@ -0,0 +1,182 @@
+/// This Data Structure is designed to hold properties.
```

Comment:
Since this itself isn't a data structure, we may want to reword this into:

PropertyMap is a specialization of SimpleMap for Tokens. It maps a String key to a PropertyValue that consists of a String type and a vec<u8> value.

Now as we define a standard here, we can circle back and say what the map of expected type is to expected value.

Reply:
Done